.travis.yml

```diff
@@ -4,6 +4,8 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.7-dev"
```

Comment:
I took a second look, and it looks like there is now an option, but it only works on Ubuntu 16.04 and requires sudo: https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905

Also, apparently 3.7-dev is old and pre-RC, so it might actually be worse than nothing. More discussion here: https://github.com/travis-ci/travis-ci/issues/9815

Do we want to use 16.04 for the whole build? Not super familiar with Travis, but I can look into build matrix options if we want to keep testing on 14.04 as well; or it looks like we can just use 16.04 for the 3.7 build.